tests/plugins/uploadimage/manual/clipboardintegration.md

```diff
@@ -0,0 +1,16 @@
+@bender-tags: 4.20.1, bug, 5333
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, uploadwidget, uploadimage, image, floatingspace, toolbar, sourcearea
+
+1. Open the console.
+
+**Expected** There is `[CKEDITOR] Error code: clipboard-image-handling-disabled.` warning in the console after the editor is loaded.
+
+**Unexpected** There is no warning in the console.
+2. Drag and drop an image into the editor.
+
+**Expected** The original name of the file is displayed above the editor.
+
+**Unexpected** The name in the `image-<timestamp>-<n>.<ext>` format is displayed above the editor.
+
+3. Repeat step `2` for the paste method.
```

Comment:
Will paste actually work? For me, it embeds the original image URL. Not sure if that case wouldn't require configuring the upload adapter with dialog:
<img width="531" alt="image" src="https://user-images.githubusercontent.com/10279586/200802470-a2648320-29f5-4492-afa0-9f2fc27ceb32.png">

Reply:
It works for me on macOS 🤔 But AFAIR it could work differently on other OSes.